python/example_code/glue/README.md

```diff
@@ -0,0 +1,116 @@
+# AWS Glue code examples for the AWS SDK for Python
+
+## Overview
+
+Shows how to use the AWS SDK for Python (Boto3) to manage AWS Glue resources.
+
+*AWS Glue is a serverless data-preparation service for extract, transform, and load 
+(ETL) operations.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+* [Create a crawler](glue_wrapper.py)
+* [Create a job definition](glue_wrapper.py)
+* [Delete a crawler](glue_wrapper.py)
+* [Delete a database from the &GLUDC;](glue_wrapper.py)
+* [Delete a job definition](glue_wrapper.py)
+* [Delete a table from a database](glue_wrapper.py)
+* [Get a crawler](glue_wrapper.py)
+* [Get a database from the &GLUDC;](glue_wrapper.py)
+* [Get a job run](glue_wrapper.py)
+* [Get runs of a job](glue_wrapper.py)
+* [Get tables from a database](glue_wrapper.py)
+* [List job definitions](glue_wrapper.py)
+* [Start a crawler](glue_wrapper.py)
+* [Start a job run](glue_wrapper.py)
+
+### Scenarios
+
+* [Get started running crawlers and jobs](scenario_getting_started_crawlers_and_jobs.py)
+
+## Running the examples
+
+### Get started running crawlers and jobs scenario
+
+This interactive scenario runs at a command prompt and shows you how to use 
+AWS Glue to:
```

Comment:
This interactive scenario runs at a command prompt and shows you how to use 
AWS Glue to do the following:

Reply:
fixed